tests/openwrt/test_dialup.py

```diff
@@ -32,11 +32,11 @@ class TestDialup(unittest.TestCase, _TabsMixin):
 """
 
     def test_render_dialup_interface(self):
-        result = OpenWrt(self._dialup_interface_netjson).render()
+        result = OpenWrt(self._dialup_interface_netjson, dsa=False).render()
```

Comment:
I don't feel very comfortable in adding `dsa=False` to all the tests, which would mean we would be testing the legacy version of the syntax everywhere.

I think we should proceed this way:

- make sure the new syntax applies only to intefaces of the following types:
  - loopback
  - ethernet
  - bridge
- all other interfaces should be able to work without the new device syntax and hence the tests can remain mostly unchanged

Reply:
Updated